icons/activity.svg

```diff
@@ -9,5 +9,5 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M22 12h-4l-3 9L9 3l-3 9H2" />
+  <path d="M22 12h-3.28a1 1 0 0 0-.948.684l-2.398 8.475a.5.5 0 0 1-.948 0L9.474 2.841a.5.5 0 0 0-.948 0l-2.298 8.475a1 1 0 0 1-.949.684L2 12.04" />
```

Comment:
Why not add an extra 1px of rounding at the base?

![image](https://github.com/lucide-icons/lucide/assets/17746067/3aec3159-5367-472b-bc38-88d7cc696e21)

```suggestion
  <path d="M22 12h-2.48a2 2 0 0 0-1.93 1.46l-2.35 8.36a.25.25 0 0 1-.48 0L9.24 2.18a.25.25 0 0 0-.48 0l-2.35 8.36A2 2 0 0 1 4.49 12H2" />
```